src/validation.h

```diff
@@ -131,6 +131,7 @@ static const int64_t MAX_FEE_ESTIMATION_TIP_AGE = 3 * 60 * 60;
 /** Default for -permitbaremultisig */
 static const bool DEFAULT_PERMIT_BAREMULTISIG = true;
 static const bool DEFAULT_CHECKPOINTS_ENABLED = true;
+static const bool DEFAULT_TEST_CNBVALID = false;
```

Comment:
This seems like it should just be a debug option that's off by default since it's something that should never be needed unless there's an internal bug.